crates/matrix-sdk-ui/Cargo.toml

```diff
@@ -61,6 +61,7 @@ matrix-sdk = { workspace = true, features = ["testing", "sqlite"] }
 matrix-sdk-test = { workspace = true }
 stream_assert = { workspace = true }
 tempfile = { workspace = true }
+url = { workspace = true }
```

Comment:
uh, can we make this dependency optional, only active if compiling docs? It seems a bit wasteful to require it overall just for a doc example.

Reply:
I mean sure, but not really easily as you would need to enable it for `cfg(doc)` but not otherwise. Or you'd need to create a doc feature.

Doesn't seem worth it since that dependency will be compiled as part of the workspace or more directly as part of the `matrix-sdk` crate which the `matrix-sdk-ui` crate directly depends on.

Reply:
Since this doesn't have a blocking emoji sign, I'll resolve this thread without doing anything.